### ISSUE TITLE
Invalid computation of productivity in PoS

### ISSUE THREAD

**Issue by shuse2** [LiskHQ/lisk-sdk#8216]
### Expected behavior
While generating a block, there is a case it try to update the productivity of PoS with invalid timestamp

### Actual behavior
```
Mar  1 18:10:49 tor1-alphanet-000 lisk-core[1352]: 2023-03-01T18:10:49.545Z DEBUG tor1-alphanet-000 application 1352 [round=24] Updating validators productivity for round 
Mar  1 18:10:49 tor1-alphanet-000 lisk-core[1352]: 2023-03-01T18:10:49.547Z FATAL tor1-alphanet-000 application 1352 [err=End timestamp must be greater than start timestamp.] System error: unhandledRejection 
Mar  1 18:10:49 tor1-alphanet-000 lisk-core[1352]: 2023-03-01T18:10:49.548Z INFO tor1-alphanet-000 application 1352 [errorCode=1 message=End timestamp must be greater than start timestamp.] Application shutdown started
```

### Steps to reproduce
Unknown

### Which version(s) does this affect? (Environment, OS, etc...)
6.0.0-alpha.15

**Comment by mosmartin** [comment]
Hi @shuse2, I can see from the logs above that these are `lisk-core` logs but you mention the env as `6.0.0-alpha.15`, which lisk-core version is running SDK 6.0.0-alpha.15? Maybe lisk-core@4.0.0-alpha.14?

**Comment by mosmartin** [comment]
For some unknown reason, I cannot reproduce this issue.

**Comment by shuse2** [comment]
This issue is resolved in the 6.0.0
